mineral-ui/mineral-ui
# [All] Adjust base font size

### Description
- Change `fontSize_base` from 8 to 16
- Fix a couple component style issues that resulted
- Update `borderRadius_1` theme value and remove unused `borderRadius_2-5` variables (can be pulled into a separate PR)
- Remove unused theme variables (can be pulled into a separate PR)

### Motivation and context

The `fontSize_base` of 8 was chosen for two reasons:

1. The design system is built on an 8px grid
1. It reduced the significant digits in our theme values and so reduced the risk of browser rounding differences

However, setting the root-level font-size to 8px for all components also has some issues. Most notably, this is what it looks like if you place a paragraph of text inside a Card (with no CardBlock):

<img width="474" alt="Tiny card content" src="https://user-images.githubusercontent.com/486540/29042773-91f16254-7b75-11e7-9c07-b6edcccb217e.png">

Using a `fontSize_base` of 16 instead would be much more expected by the consumer, as it is the default browser font-size. Additionally, each of the two reasons above can be addressed:

1. 16 is simply double the 8px grid value; the mental math isn’t too much harder
1. browser rounding differences can be mitigated by applying a strict (i.e. set heights & line-heights) layout strategy, which is likely necessary anyway (I confirmed this is the case for Button; no visual changes in either Edge or IE11).

### How to test

1. `cd mineral-ui/packages/site && npm start`
2. Confirm all components look as they should



### Types of changes

- Breaking change (fix or feature that would cause existing functionality to change)

### Checklist




### How does this PR make you feel?

![Algorithms all around me](https://media.giphy.com/media/AXorq76Tg3Vte/giphy.gif)
